includes/options/class-amp-options-manager.php

```diff
@@ -562,6 +571,63 @@ public static function persistent_object_caching_notice() {
 		}
 	}
 
+	/**
+	 * Render the Stories deprecation admin notice.
+	 */
+	public static function render_stories_deprecation_notice() {
+		if (
+			self::is_stories_experience_enabled() &&
+			(
+				'edit-amp_story' === get_current_screen()->id ||
+				'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id
+			)
+		) {
+			echo '<div class="notice notice-warning"><p>';
+			printf(
+				'%s %s %s',
+				esc_html__( 'The Stories experience is being extracted from the AMP plugin into a separate standalone plugin which will be available soon. Please backup or', 'amp' ),
+				sprintf(
+					'<a href="%s">%s</a>',
+					esc_url( 'https://amp-wp.org/documentation/amp-stories/exporting-stories/' ),
+					esc_html__( 'export your existing Stories', 'amp' )
+				),
+				esc_html__( 'as they will not be available in the next version of the AMP plugin.', 'amp' )
+			);
+			echo '</p></div>';
```

Comment:
Please refactor the warning to match the warning in the editor:

![image](https://user-images.githubusercontent.com/134745/73989335-4da67580-48fa-11ea-9439-b762e442b04c.png)

This will reduce the strings to translate.

Aside: Breaking up the strings in this way would make it hard to translate in some languages.

Reply:
Good point, made the change in 8c14de5c2a84021d6921960a70039c109a7f4398.